lib/cs_guide/auth.ex

```diff
@@ -21,4 +21,12 @@ defmodule CsGuide.Auth do
     |> Plug.Conn.put_session(:user_id, user.entry_id)
     |> Plug.Conn.configure_session(renew: true)
   end
+
+  def venue_owner(conn, venue) do
+    current_venue_ids = conn.assigns[:venue_id] || ""
+    venue_ids = current_venue_ids <> "," <> venue.entry_id
```

Comment:
I'm getting an argument error on this line because `current_venue_ids` is defined as a list [here](https://github.com/club-soda/club-soda-guide/pull/241/files#diff-233aca7b8209169fc0379065e95f4ff9R11)

Reply:
I think we should just define it as a string. Users can only sign up with one venue, so a list shouldn't be necessary